pages/instructorAssessmentDownloads/instructorAssessmentDownloads.js

```diff
@@ -291,6 +292,22 @@ router.get('/:filename', function(req, res, next) {
             if (ERR(err, next)) return;
             archive.finalize();
         });
+    } else if (req.params.filename == res.locals.groupConfigs) {
```

Comment:
"groupConfigs" is a misleading name for this since it's actually just the list of group memberships